Makefile

```diff
@@ -9,6 +9,16 @@ BOX_URL="https://github.com/humbug/box/releases/download/3.1.0/box.phar"
 PHP-CS-FIXER_URL="https://cs.sensiolabs.org/download/php-cs-fixer-v2.phar"
 PHPSTAN_URL="https://github.com/phpstan/phpstan/releases/download/0.10.3/phpstan.phar"
 
+TEST_TRAVIS_OPTIONS ?= false
+ifeq ($(TEST_TRAVIS_OPTIONS),false)
```

Comment:
Wouldn't it be better to do some if not eq to false, instead of an empty branch and then an else statement?
or even  `ifeq ($(TEST_TRAVIS_OPTIONS),true)` ?